docs_src/microservices/device/supported/device-usb-camera/supplementary-info/advanced-options.md

```diff
@@ -1,5 +1,34 @@
 # Advanced Options
 
+## RTSP Auth
+The device service allows for rtsp stream authentication using the [rtsp-simple-server](https://github.com/aler9/mediamtx). Authentication is enabled by default.
+
+To configure the username and password for the authentication when buiding your own images, edit the fields in the 'configuration.yaml'
+!!! note 
+    Leaving the fields blank will **NOT** disable authentication. The stream will not be able to be authenticated until credentials are provided.
+
+!!! example - "Snippet from configuration.yaml"
+    ```yaml
+    ...
+    Writable:
+        LogLevel: "INFO"
+        InsecureSecrets:
+            rtspauth:
+            SecretName: rtspauth
+            SecretData:
+                username: ""
+                password: ""
+    ```
+
+To disable authentication, comment out the externalAuthenticationURL line in the `Dockerfile`.
+!!! example - "Snippet from the Dockerfile"
+    ```Dockerfile
+    # disable unused rtsp-simple-server listeners
+    ...
+    # RUN sed -i 's,externalAuthenticationURL:,externalAuthenticationURL: http://localhost:8000/rtspauth,g' rtsp-simple-server.yml
```

Comment:
Should we just include this line so that the user does not get confused on what to comment out?

Reply:
I was unsure, because I think that including the comment above shows the users the heading that this is under, but I don't think it is super necessary

Reply:
either one is fine for me. also maybe let them know it can be configured to a different port here.